manager-bundle/src/HttpKernel/JwtManager.php

```diff
@@ -58,13 +47,40 @@ public function __construct(string $projectDir, Signer $signer = null, Builder $
         $secretFile = $projectDir.'/var/jwt_secret';
 
         if ($filesystem->exists($secretFile)) {
-            $this->secret = file_get_contents($secretFile);
+            $secret = file_get_contents($secretFile);
+        }
+
+        if (!\is_string($secret) || 64 !== \strlen($secret)) {
+            $secret = bin2hex(random_bytes(32));
+            $filesystem->dumpFile($secretFile, $secret);
+        }
+
+        if (null !== $signer) {
+            @trigger_error('Second argument ($signer) of JwtManager::__construct(...) is deprecated since Contao 4.9, to be removed in Contao 5.0. Use the Configuration object instead.', E_USER_DEPRECATED);
```

Comment:
Since the class is internal, we do not have to keep the old constructor arguments.

Reply:
👍 I'll remove it.

Reply:
Fixed in 2f4a70de1.